src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -187,11 +189,25 @@ QString AccessibleItem::accessibleName() const
     QString commandInfo = root ? root->commandInfo() : "";
     QString staffInfo = root ? root->staffInfo() : "";
     QString barsAndBeats = m_element->formatBarsAndBeats();
+    Selection selection = m_element->score()->selection();
+    QString selectionText = "";
+
+    if (selection.isRange() && selection.endSegment()) {
+        Segment* prevSeg = selection.endSegment()->prev1(SegmentType::ChordRest);
+        if (m_element->tick() != selection.tickStart() || m_element->tick() != prevSeg->tick()) {
+            if (m_element->tick() == selection.tickStart()) {
+                selectionText = "Selection begins";
+            } else if (prevSeg && m_element->tick() == prevSeg->tick()) {
+                selectionText = "Selection ends";
```

Comment:
Not translatable?

Reply:
Its just spoken out loud. So I assumed there was no need for it

Reply:
It should be translated like the rest of the UI, otherwise the button names would be in one language and the selection messages in a different one. The screen reader will just read whatever it is told to read, so it's up to us to translate it.

Reply:
Ah okay. I didn't know that